blocks/default-output-block/output-types/default.jsx

```diff
@@ -52,19 +52,56 @@ const googleTagManagerNoScript = (gtmID) => {
   );
 };
 
+const optimalFontLoading = (fontUrl, index = '') => (
+  <>
+    <link
+      rel="preload"
+      as="style"
+      href={`${fontUrl}&display=swap`}
+    />
+    <link
+      rel="stylesheet"
+      key={fontUrl}
+      data-testid={`font-loading-url-${index}`}
+      href={`${fontUrl}&display=swap`}
+    />
+
+    <noscript>
```

Comment:
This `<noscript>` is doing nothing more or different to the `link` on line 62 - This technique is only used when you set stylesheets to be a print media by default and then switch them to all with onload technique

Reply:
ah. that makes sense. I saw the onload technique but couldn't validate that switch working as expected. I'll remove the noscript

Reply:
ah. that makes sense. I saw the onload technique but couldn't validate that switch working as expected. I'll remove the noscript